deeppixel/iqa/README.md

```diff
@@ -0,0 +1,72 @@
+# This module(brisque) deals with:
+- Assessing the Image Quality Scores 
+- Blur_Detection in the images that we upload/refer a path to.
+
+**To import this module simple type:**
+from deeppixel import iqa.brisque as b
+
+**To Read an Image from local computer:**
+img_color,img_gray = b.imgread( #path to the image file)
+*Note that It returns 2 images, one is color(RGB) and the other one is grayscale.*
+
+**To Read an Image from URL:**
```

Comment:
Can you please shift this to the main readme file or inside the test folder as I requested